src/api/updateUserDetails.ts

```diff
@@ -0,0 +1,19 @@
+// utils/updateUserDetails.js
+import axios from 'axios';
+
+const updateUserDetails = async (formData: any) => {
+  try {
+    const response = await axios.put('https://carrerhub-backend.vercel.app/api/v1/user/details', formData, {
```

Comment:
Donot hardcode the urls there like this, later on we have to change the url so ge the url from env, if you don't have the env you can ask for it. First add the url to api/index.ts and the use it here